tekton/publish.yaml

```diff
@@ -80,7 +80,7 @@ spec:
       cp ${DOCKER_CONFIG} /workspace/docker-config.json
 
   - name: create-ko-yaml
-    image: golang:1.18.7
+    image: cgr.dev/chainguard/go:latest-dev
```

Comment:
I think we need a version here instead of `latest-dev`

Reply:
Versioned tags aren't part of free tier. Pinned the digest + enabled dependabot!